src/Sandbox.js

```diff
@@ -330,12 +367,12 @@ function packageInfoKey(env: Environment, packageInfo: PackageInfo) {
       },
     });
     if (process.env.ESY__TEST) {
-      packageInfo.__key = `${normalizedName}-${version || '0.0.0'}`;
+      packageInfo.__cachedPackageHash = `${normalizedName}-${version || '0.0.0'}`;
```

Comment:
Will the env already be fully taken into account in the `hash({})` above which accepts `env`?